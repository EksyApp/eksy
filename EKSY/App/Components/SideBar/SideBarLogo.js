import React, {Component} from 'react'
import {View, StyleSheet} from "react-native";
import Picture from "../Common/Picture";

class SideBarLogo extends Component {
	
	constructor(props) {
		super(props)
		
		this.data = {
			width: 185,
			height: 54,
			uri: ""
		}
	}
	
	render() {
		return (
				<Picture source={require("../../Images/logoPlaceHolder.png")} data={this.data} containerStyle={styles.view}/>
		)
	}
}

const styles = StyleSheet.create({
	view: {
		width: '100%',
		marginBottom: 20
	}
})

export default SideBarLogo
