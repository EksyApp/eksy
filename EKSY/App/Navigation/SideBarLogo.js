import React, {Component} from 'react'
import {View, Text, Image} from 'react-native'
import Style from './Styles/SideBarStyles'

class SideBarLogo extends Component {

  render() {
    return(
      <Image source={require('../Images/logoPlaceHolder.png')} style={Style.image} />
    )
  }
}

export default SideBarLogo
