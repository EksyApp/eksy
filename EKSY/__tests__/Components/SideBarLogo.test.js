import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import SideBarLogo from "../../App/Components/SideBar/SideBarLogo";

test('SideBarLogo renders correctly', () => {
	const tree = renderer.create(
			<SideBarLogo />
	).toJSON();
	expect(tree).toMatchSnapshot();
});