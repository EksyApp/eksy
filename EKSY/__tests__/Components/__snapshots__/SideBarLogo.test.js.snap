// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarLogo renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      undefined,
      Object {
        "marginBottom": 20,
        "width": "100%",
      },
      Object {
        "height": 54,
        "width": "100%",
      },
    ]
  }
>
  <Image
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    resizeMode="cover"
    source={1}
    style={
      Array [
        Object {},
        Object {
          "height": 54,
          "width": "100%",
        },
      ]
    }
    testID={undefined}
  />
</View>
`;
